### PR TITLE
Add home mode 4 to mclennan

### PR DESCRIPTION
**Pull request by Tom-Willemsen**
https://github.com/ISISComputingGroup/IBEX/issues/5729

### CODE REVIEW

File: motorApp/MclennanSrc/devPM304.cc [hunk 1]
```diff
@@ -164,10 +164,12 @@ STATIC void request_home(char* buff, int model, int axis, int home_direction, in
     if (model == MODEL_PM304){
         sprintf(buff, "%dIX%d;", axis, home_direction);
     } else {
-        int motor_default=0, constant_velocity=1, reverse_and_zero=2;
+        int motor_default=0, constant_velocity=1, reverse_and_zero=2, constant_velocity_move_and_zero=3, forward_and_zero=4;
         if ( home_mode==motor_default || home_mode==reverse_and_zero ) {
```
Comment (John-Holt-Tessella) [inline motorApp/MclennanSrc/devPM304.cc]:
```suggestion
        if ( home_mode==motor_default || home_mode==reverse_and_zero || home_mode==forward_and_zero) {
```

File: motorApp/MclennanSrc/devPM304.cc [hunk 2]
```diff
@@ -164,10 +164,12 @@ STATIC void request_home(char* buff, int model, int axis, int home_direction, in
     if (model == MODEL_PM304){
         sprintf(buff, "%dIX%d;", axis, home_direction);
     } else {
-        int motor_default=0, constant_velocity=1, reverse_and_zero=2;
+        int motor_default=0, constant_velocity=1, reverse_and_zero=2, constant_velocity_move_and_zero=3, forward_and_zero=4;
```
Comment (John-Holt-Tessella) [inline motorApp/MclennanSrc/devPM304.cc]: Can we use the defines from the homing st? or repeat them in this file rather than this?

**Review by John-Holt-Tessella — DISMISSED** [review]
Update documentation including home modes in https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/McLennan-motors

**Review by DominicOram — APPROVED** [review]
Looks good, thanks

**Pull request merged**